server/modules/tasks/server/models/sequelize/comments.server.model.sequelize.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+module.exports = function(sequelize, DataTypes) {
```

Comment:
What is `sequelize` here? Why is it necessary to pass into here?

If it's an instance of the Sequelize library, I'm guessing that it's needed because the library doesn't expose a singleton the way that Mongoose does after the first require. Is that right?